.github/ISSUE_TEMPLATE/issue_API_description_template.md

```diff
@@ -0,0 +1,17 @@
+---
+name: ❕ API_description 📝
+about: Indicate an issue with an API description at the CAMARA website
+title: 'API description change request'
+labels: 'API_description'
+assignees: ''
+
+---
+
+**Requested change**
+* Change of API description: <!-- Y/N  -->
```

Comment:
That's not clear enough, as already the title is "API description change request", so I would expect here always a "yes"